reviewing/license-scanner/src/getPkgInfo.ts

```diff
@@ -126,10 +140,18 @@ async function parseLicense (
       } else {
         licenseContents = await readLicenseFileFromCafs(opts.cafsDir, licensePackageFileInfo as PackageFileInfo)
       }
+      const licenseContent = licenseContents?.toString('utf-8')
+      let name = 'Unknown'
+      if (licenseContent) {
+        const match = new RegExp(LICENSE_NAMES.join('|'), 'i').exec(licenseContent)
```

Comment:
this will match "mit" when found inside a different word. Let's not ignore case and only match separate words. I think this should work
```suggestion
        const match = new RegExp(`\b(${LICENSE_NAMES.join('|')})\b`, 'i').exec(licenseContent)
```